package.json

```diff
@@ -27,6 +27,7 @@
   "dependencies": {
     "@headlessui/react": "^1.3.0",
     "@heroicons/react": "1.0.2",
+    "@martinstark/throttle-ts": "^1.2.3",
```

Comment:
Please install the new dependencies using `yarn`, so that the `yarn.lock` file would be updated as well and the CI check can pass.

Reply:
@zainfathoni ok got it, thanks for the advice